hbw/app/javascript/packs/hbw/components/form.js.jsx

```diff
@@ -216,9 +216,38 @@ modulejs.define('HBWForm', ['React', 'jQuery', 'HBWError', 'HBWFormDatetime',
         return result;
       }, {});
 
-      this.setState({ formValues: variables });
+      this.setState({ formValues: this.updateSelectVariables(variables) });
     };
 
+    updateSelectVariables = (variables) => {
+      const flattenFields = this.flatFields(this.props.form.fields);
+
+      const fieldNamesforUpdate = flattenFields.filter(
+        field => field.type === 'select' && field.mode === 'select' && field.nullable === false && field.choices.length
+      ).map(field => field.name);
+
+      fieldNamesforUpdate.forEach(((fieldName) => {
+        const updatingField = flattenFields.find(field => field.name === fieldName);
+        let value = updatingField.choices[0];
+
+        if (Array.isArray(value)) {
+          [value] = value;
```

Comment:
as for me, then it's better to use [0]

Reply:
I tried, but linter shows `Use array destructuring (eslintprefer-destructuring)`